exercises/saddle-points/example.js

```diff
@@ -1,27 +1,30 @@
-function findSaddlePoints(rows, rowMaxs, colMins) {
-  return rows.reduce((saddlePoints, row, rowIndex) => {
-    row.forEach((cell, colIndex) => {
-      if (cell === rowMaxs[rowIndex] && cell === colMins[colIndex]) {
-        saddlePoints.push([rowIndex, colIndex]);
+export class SaddlePoints {
```

Comment:
Good that you removed `Matrix` class & old tests. Here I would suggest using a plain function instead of static method inside class. In https://github.com/exercism/javascript/issues/274 we decided to use classes only if some local state is required.